packages/social-trader/src/components/notifications/components/notification/notification.tsx

```diff
@@ -84,6 +85,12 @@ const AssetAvatarContainerStyle = css<IAssetAvatarContainerProps>`
     }
     return $labelColor;
   }};
+
+  img {
```

Comment:
Don't use specificity. Use props and affect to img styles directly.

Reply:
Do i need to pass into AssetAvatar hasStaticUrl prop? It has a pretty big nesting

Reply:
Yes.